consensus/consensus.go

```diff
@@ -514,13 +514,13 @@ func (cs *consensus) HandleQueryVote(height uint32, round int16) *vote.Vote {
 	votes := []*vote.Vote{}
 	switch {
 	case round < cs.round:
-		// Past round: Only broadcast cp:decided votes
-		vs := cs.log.CPDecidedVoteSet(round)
+		// Past round: broadcast cp:decided votes for the previous round.
+		vs := cs.log.CPDecidedVoteSet(cs.round - 1)
```

Comment:
@b00f can you more explain it, why broadcast for the previous round?

Reply:
Please check the comment again...